docs/src/components/HomepageFeatures.js

```diff
@@ -1,3 +1,19 @@
+/*
```

Comment:
some double license headers here, is this file apache or cc-by-sa?

Reply:
It's kinda both, honestly. The 'Copy' in it is something that we probably don't want to license under Apache or else people will be able to impersonate our site legally.

All of the Markdown files in the repo are CC-BY-SA for this reason -- if somebody forks it they have to state that it's a fork and specify their changes. That's important for content like docs.

For our normal code, we just need Apache or BSL.

But for our docs -- where code is mixed with content -- we probably need both licenses. (I don't know how CC-BY-SA works for code, and I don't want to rely on it exclusively. Apache does a better job at that.)

If somebody asks us about this in the future because they want to fork our docs, we can discuss it with them.

Summary of CC-BY-SA:
![image](https://user-images.githubusercontent.com/4573221/167230587-8dc0b46a-1555-4c58-90b7-a1ba7fa7d7be.png)